### PR TITLE
Update plotIt documentation

### PR DESCRIPTION
**Pull request by pieterdavid**
I picked up / added documentation for:
- the recent additions by @swertz to the [wiki](https://github.com/cp3-llbb/plotIt/wiki/List-of-configuration-options) (to be replaced by a link here)
- https://github.com/cp3-llbb/plotIt/pull/112 (`ratio-style`)
- https://github.com/cp3-llbb/plotIt/pull/115 (eras support)
- https://github.com/cp3-llbb/plotIt/pull/113 (cmake build)

### CODE REVIEW

File: docs/plotit.md [hunk 1]
```diff
@@ -25,6 +25,16 @@ cd ..
 make -j 4
 ```
 
+Outside CMSSW and with a recent version of ROOT, plotIt can als be built with CMake (and an install prefix passed with `-DCMAKE_INSTALL_PREFIX`, or the exectuble copied from the build directory to install), e.g.
+```bash
+git clone -o upstream https://github.com/cp3-llbb/plotIt.git
+mkdir plotit-build
+cd plotit-build
+cmake ../plotit
```
Comment (swertz) [inline docs/plotit.md]: Does that work? Shouldn't it be `cmake ..`?

Comment (pieterdavid) [reply docs/plotit.md]: in this case `plotIt` and `plotit-build` are next to one another, so I think so (but there is a typo)

Comment (swertz) [reply docs/plotit.md]: Oh right you're not moving into `plotIt`... I'm used to create the build dir inside the project dir. Fine then.

**Comment by swertz** [comment]
Thanks!

**Pull request merged**